### PR TITLE
Update README. Remove border from Long Description accordion

### PR DESCRIPTION
**Pull request by sirleech**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,31 +11,29 @@
 
 ## Staging
 
-- **url**: [https://service-manual.apps.staging.digital.gov.au/](https://service-manual.apps.staging.digital.gov.au/)
-- **branch**: `develop`
-- **status**:  [![CircleCI](https://circleci.com/gh/govau/service-manual/tree/develop.svg?style=svg)](https://circleci.com/gh/govau/service-manual/tree/develop)
```
Comment (alex-page) [inline README.md]: The url and status shouldn't be removed

File: README.md [hunk 2]
```diff
@@ -11,31 +11,29 @@
...
+- **branch**: `test-staging`
 
 
 ## Development
 
-❗️All pull requests should be compared against `develop`
```
Comment (alex-page) [inline README.md]: This is the correct comment, don't change it. Test-staging is a test branch which will likely be removed one day.

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Service Manual
+# Digital Guides
```
Comment (alex-page) [inline README.md]: Think about renaming the repository aswell

File: README.md (hunk 2 above)
Comment (sirleech) [reply README.md]: Will fix

File: README.md (hunk 1 above)
Comment (alex-page) [reply README.md]: Wait until you have a new application set up in cloud foundry and urls approved etc. before doing these changes.

Comment (sirleech) [reply README.md]: Might just revert it back

**Pull request merged**